docs/spec/v2alpha1/helmreleases.md

```diff
@@ -1,504 +1,125 @@
 # Helm Releases
 
-The `HelmRelease` API defines a resource for automated controller driven Helm actions.
+The `HelmRelease` CRD defines a resource for controller driven reconciliation
+of Helm releases via Helm actions such as install, upgrade, test, uninstall, and rollback.
+This includes release placement (namespace/name), release content (chart/values overrides),
+action trigger configuration, individual action configuration, and statusing.
 
-## Specification
+## Helm release placement
 
-A **helmrelease** object defines the source of a Helm chart by referencing an object
-managed by [source-controller](https://github.com/fluxcd/source-controller), the interval
-reconciliation should happen at, and a set of options to control the settings of the
-automated Helm actions that are being performed.
+The namespace/name in which to deploy the helm release defaults to the namespace/name of the
```

Comment:
```suggestion
The namespace/name in which to deploy the Helm release defaults to the namespace/name of the
```

As long as we are not directly referring to the binary, the name should be capitalized.